vyper/parser/parser.py

```diff
@@ -800,7 +1012,7 @@ def pack_logging_data(expected_data, args, context, pos):
 
     # Unroll any function calls, to temp variables.
     prealloacted = {}
-    for idx, (arg, expected_arg) in enumerate(zip(args, expected_data)):
+    for idx, (arg, _expected_arg) in enumerate(zip(args, expected_data)):
```

Comment:
`_expected_arg` doesn't seem to be used use `_` instead.

Reply:
I had thought to just use the underscore prefix in case someone wonders what is being ignored. Using the prefix tells flake8 to ignore it.

Reply:
Ah, OK that works too :+1: